shell/platform/windows/platform_handler.cc

```diff
@@ -226,10 +226,17 @@ PlatformHandler::~PlatformHandler() = default;
 void PlatformHandler::GetPlainText(
     std::unique_ptr<MethodResult<rapidjson::Document>> result,
     std::string_view key) {
+  const FlutterWindowsView* view = engine_->view();
+  if (view == nullptr) {
+    result->Error(kClipboardError,
+                  "Clipboard is not available in Windows headless mode");
```

Comment:
This message mirrors the GLFW message: 

https://github.com/flutter/engine/blob/f75287af0b19c07d5529e5c990f7e4c1740f50c8/shell/platform/glfw/platform_handler.cc#L47-L51

Please let me know if you have suggestions though!